integration/ctr/oci_hooks_tests.bats

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bats
```

Comment:
You're missing a copyright statement here.
Please, check https://github.com/kata-containers/tests/blob/8fa30acf6d5a2f1f137a6e47c4de21c45dc9b2e3/integration/kubernetes/k8s-inotify.bats#L1-L7 for instance